pdns/dnsdistdist/configure.ac

```diff
@@ -8,7 +8,7 @@ AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
 AC_PROG_CXX
 AC_LANG([C++])
-AC_GNU_SOURCE
+LT_INIT()
```

Comment:
Don't we have that a few lines below already?

Reply:
indeed, fixing